src/resources/time-sheets.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import fs from 'fs';
+
+const router = express.Router();
+const timeSheets = require('../data/time-sheets.json');
+
+router.put('/edit/:id', (req, res) => {
+  const found = timeSheets.some((timeSheet) => timeSheet.id === (req.params.id));
+  if (found) {
+    const upTS = req.body;
+    timeSheets.forEach((timeSheet) => {
+      if (timeSheet.id === req.params.id) {
+        const tSheet = timeSheet;
+        tSheet.task = upTS.task ? upTS.task : tSheet.task;
```

Comment:
Let's take the same approach as in the next lines,
```suggestion
        tSheet.task = upTS.task ?? tSheet.task;
```